### PR TITLE
chore(liveness): move liveness example app to its own app (#4451)

### PR DESCRIPTION
**Pull request by thaddmt**
* chore(liveness): move liveness example app to its own app

* address feedback

* uppgrade outdated deps

* Update examples/next-liveness/.eslintrc

---------

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by calebpollman — COMMENTED** [review]
In terms of the code/example app, left some minor questions around configuration/deps but nothing major. Had issues running the React Native app when i checked out and ran locally though, can you sanity check?

File: examples/next-liveness/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "@aws-amplify/ui-react-liveness": "^2.0.6",
+    "@aws-sdk/credential-providers": "^3.370.0",
+    "aws-amplify": "5",
+    "next": "^13.4.19",
+    "next-global-css": "^1.1.1",
+    "react": "18",
+    "react-copy-to-clipboard": "^5.1.0",
+    "react-dom": "18",
+    "react-icons": "^4.4.0",
+    "react-map-gl": "7.0.23",
```
[the remark this answers is not in the document]
Comment (thaddmt) [reply examples/next-liveness/package.json:23]: nope good catch, will remove

File: examples/next/tsconfig.json [hunk 2]
```diff
@@ -19,8 +19,11 @@
     "jsx": "preserve",
     "baseUrl": "./",
     "paths": {
-      "@environments/*": ["../../environments/*"]
-    }
+      "@environments/*": [
+        "../../environments/*"
+      ]
+    },
+    "incremental": true
```
[the remark this answers is not in the document]
Comment (thaddmt) [reply examples/next/tsconfig.json:26]: think this may have automatically been added when upgrading next, im not sure if it's needed will check

**Comment by thaddmt** [comment]
@calebpollman tested on this branch and rn android seemed to work 
Screenshot 2023-09-27 at 11 11 48 AM

**Comment by thaddmt** [comment]
Closing this as we are going to maintain liveness v5 on a separate branch

**Pull request closed without merging**